gradle.properties

```diff
@@ -53,7 +53,7 @@ IS_MINIFY_APP_ENABLED=true
 ZCASH_DEBUG_KEYSTORE_PATH=
 
 # Versions
-ANDROID_MIN_SDK_VERSION=16
+ANDROID_MIN_SDK_VERSION=19
```

Comment:
This would drop support for Android that are over 10 years old, which represents less than 1% of all Android devices.  [Google Play Services requires API 19](https://android-developers.googleblog.com/2021/07/google-play-services-discontinuing-jelly-bean.html), which is a gating factor for apps consuming our SDK distributed through Google Play.

The context is that this line is a fix for regression introduced recently.  GRPC now has a minimum of API 19.  Our options are either:
 - Roll back to an older version of GRPC
 - Keep newer GRPC, and bump our SDK to API 19